src/main/java/bio/terra/service/resourcemanagement/google/GoogleBucketService.java

```diff
@@ -280,12 +280,16 @@ private Bucket newCloudBucket(GoogleBucketResource bucketResource) {
     GoogleProjectResource projectResource = bucketResource.getProjectResource();
     String googleProjectId = projectResource.getGoogleProjectId();
     GcsProject gcsProject = gcsProjectFactory.get(googleProjectId);
```

Comment:
These aren't needed until after the code creates `bucketInfo` so it might be clearer to define them there instead of here.